integration_tests/run.sh

```diff
@@ -26,17 +20,37 @@ else
 fi
 swbundle_name="${prefix}2grid-swbundle-${end}"
 
-./create_conda_software_bundle.sh "${WORKSPACE}/${swbundle_name}"
-export POLAR2GRID_HOME="$WORKSPACE/$swbundle_name"
-cd "$WORKSPACE/integration_tests"
+# Activate conda for bash.
+/data/users/davidh/miniconda3/bin/conda init bash
+# Restart the shell to enable conda.
+source ~/.bashrc
+
 # Documentation environment also has behave, while the build environment does not.
+conda env update -n jenkins_p2g_docs -f "$WORKSPACE/build_environment.yml"
 conda env update -n jenkins_p2g_docs -f "$WORKSPACE/jenkins_environment.yml"
 conda activate jenkins_p2g_docs
+pip install "$WORKSPACE"
+
+if [[ "$GIT_TAG_NAME" =~ [pg]2g-skip-tests ]]; then
```

Comment:
We don't want to make these types of decisions based on the tag, but rather on the commit message. I don't remember how jenkins makes this available `GIT_COMMIT_MSG` maybe? It's in the setting.

We don't want to make a new tag every time we want to run tests in a special way. Also there would be no way to stop the tests from running in master if it was dependent on tags. The way Travis and other CIs typically do this is something like:

```
git commit -m "My normal message [skip-ci]
```

We could use this idea to detect `[skip-tests]` or `[p2g-skip-tests]` or something.